mods/ts/rules/defaults.yaml

```diff
@@ -508,6 +508,30 @@
 		AirborneUpgrades: airborne
 	ReturnOnIdle:
 
+^AircraftHusk:
+	RenderVoxels:
```

Comment:
Don't forget `RenderSprites` too!
All actors (even voxel ones) need this to display smoke effects, etc.